Dockerfile

```diff
@@ -51,7 +52,16 @@ RUN set -ex \
     && make drafter \
     && make install \
     && drafter -v \
-    && cd && rm -rf /tmp/drafter \
+  # Install opencensus extension
+  && wget https://pecl.php.net/get/opencensus-0.2.2.tgz -O opencensus.tgz \
```

Comment:
@vyas-shyam Actually, I need some changes in this extensions that aren't released yet. Could you please build the latest `master` branch, and pin it by the commit hash?